### DISCUSSION TITLE
Montetization, What are the goals

### DISCUSSION THREAD

**Discussion by Miladiir** [omnivore-app/omnivore#4218] [General]
Hi,
I just stumbled upon this project and tried it out. I was wondering: What are the goals for this project? Is it to provide a great modern alternative to RIL services and hybrids like Readwise for free? Are there plans for monetization?

**Comment by jacksonh** [comment]
> Hi, I just stumbled upon this project and tried it out. I was wondering: What are the goals for this project? Is it to provide a great modern alternative to RIL services and hybrids like Readwise for free? Are there plans for monetization?

We'd like to offer some more premium features that could be monetized, such as expanding our text to speech, perhaps some AI integration features, and some other AI/ML-type features that are a bit more costly to run.
